.circleci/config.yml

```diff
@@ -272,9 +275,12 @@ jobs:
               time make container DOCKER_ARGS=--no-cache EXTRA_PATH=/home/linuxbrew/.linuxbrew/bin
               echo "--- Test container $dir"
               time make test EXTRA_PATH=/home/linuxbrew/.linuxbrew/bin
+              echo "--- Cross-compile container $dir"
+              time make multi-arch EXTRA_PATH=/home/linuxbrew/.linuxbrew/bin
```

Comment:
Do we want this here? Currently, the script loops through all containers one by one and then 1) makes the container, 2) runs tests and 3) cross-compiles.

I think it makes more sense to have a separate CircleCI step "Linux container multi-arch" - if one or more tests fail, CircleCI will fail early, prior to cross-compiling for other architectures (now it's only arm64, but might be more in the future). Curious to hear what you think about this.

Reply:
I'm fine with separate CircleCI step "Linux container multi-arch".